### PR TITLE
Add winget support

### PR DESCRIPTION
**Pull request by vladimyr**
[winget](https://github.com/lukesampson/scoop) is a command-line package installer for Windows. (See announcement post: https://devblogs.microsoft.com/commandline/windows-package-manager-preview/)
The app information is fetched from [community repo](https://github.com/microsoft/winget-pkgs) hosting YAML manifests and parsed using [`yaml`](https://npm.im/yaml) package.

This adds new handler:
```
/winget/:topic<v|license>/:appId
```
where the topic can be one of:
- `v` (version)
- `license`

## Preview

image

### CODE REVIEW

File: api/winget.ts [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    '/winget/v/Balena.Etcher': 'version',
+    '/winget/license/Arduino.Arduino': 'license'
+  },
+  handlers: {
+    '/winget/:topic<v|license>/:appId': handler
+  }
+})
+
+async function handler ({ topic, appId }: PathArgs) {
+  const versions = await fetchVersions(appId)
```
Comment (amio) [inline api/winget.ts:109]: This request can be moved to `case 'v': {...}`, or we will fetch version twice for license badge

Comment (vladimyr) [reply api/winget.ts]: Thank you! You are absolutely right. Ah, those joys of late-night coding 🙃

**Comment by vladimyr** [comment]
I've extracted duplicated GitHub querying logic (REST & graphql) to a new github library.

**Comment by amio** [comment]
Hello MSBuild 2020 🎉

**Pull request merged**